### PR TITLE
Added job cards to the resources listing page

### PR DESCRIPTION
**Pull request by JCArya**
Targets #1568
 
Description:
added cards related to the job and create a view of every card

### CODE REVIEW

File: resources/views/layouts/navbar.blade.php [hunk 1]
```diff
@@ -25,7 +25,7 @@
                 @endcan
                 <a class="dropdown-item" href="{{route('userappointmentslots.show',auth()->id())}}">Appointment Slots</a>
                 <a class="dropdown-item" href="{{ route('hr.tags.index') }}">{{ __('Manage Tags') }}</a>
-                <a class="dropdown-item disabled" href="{{ route('resources.index') }}">Guidelines And Resources</a>
+                <a class="dropdown-item " href="{{ route('resources.index') }}">Guidelines And Resources</a>
```
Comment (P4NK4J) [inline resources/views/layouts/navbar.blade.php:28]: Let's keep it disabled for a while, we can enable it once the page is ready to use

File: Modules/HR/Resources/views/guidelines-resources/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+@extends('hr::layouts.master')
+@section('content')
+<div class="row col-gap justify-content-center">
+    <div class="col-md-8">
+        <h3>Resources for Laravel Developer</h3>
+    </div>
+    <div class="col-md-2">
+        <a href="#" class="btn btn-success btn-block btn-lg">Add Resources</a>
```
Comment (P4NK4J) [inline Modules/HR/Resources/views/guidelines-resources/show.blade.php:8]: Let's add a comment here `//ToDo: Functionality yet to be created`

File: Modules/HR/Http/Controllers/ResourcesController.php [hunk 3]
```diff
@@ -3,11 +3,19 @@
...
 
 use Illuminate\Routing\Controller;
+use Modules\HR\Entities\Job;
 
 class ResourcesController extends Controller
 {
     public function index()
     {
-        return view('hr::guidelines-resources.resources');
+        $jobs = Job::select('title')->get();
```
Comment (P4NK4J) [inline Modules/HR/Http/Controllers/ResourcesController.php:12]: Why we are using `$jobs = Job::select('title')->get();`, what would happen if we use `all()` ?

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please take a look

File: resources/views/layouts/navbar.blade.php (hunk 1 above)
Comment (JCArya) [reply resources/views/layouts/navbar.blade.php]: okk

**Review by P4NK4J — APPROVED** [review]

**Pull request merged**